### DISCUSSION TITLE
GWT Plugins for Eclipse v3.0.0

### DISCUSSION THREAD

**Discussion by arifmasyhur20** [gwt-plugins/gwt-eclipse-plugin#473] [Q&A]
Hi everyone,

I'm new to the GWT plugin and just discovered about this while working to upgrade team current system which are still using the old Google Plugins for Eclipse with GWT SDK 2.4.0. I need to upgrade the GWT plugin to the version 3.0.0 and GWT SDK 2.9.0 as the highest to standardize with other system that use it. Does anyone know if it is still possible for me to get the plugin? The eclipse marketplace only provided the latest version as far as I know.

**Comment by protoism** [comment]
I'm not sure I understand why you need 3.0.0 but... yes, it is possible to install it

The docs currently point to version 3:
    https://gwt-plugins.github.io/documentation/gwt-eclipse-plugin/Download.html
and the plugin is here:
    http://storage.googleapis.com/gwt-eclipse-plugin/v3/release

**Reply by arifmasyhur20** [reply]
It just to standardized with our other system that use 3.0.0, I try that link, but the plugin does not show up in my eclipse, but then I tried one from this issue #406 and now its look okay. Anyway, thank you for the answer!
